src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -91,6 +91,18 @@ public static void EnsureDirectory(string directoryPath)
         /// <summary>
         /// Returns path2 relative to path1, with Path.DirectorySeparatorChar as separator
```

Comment:
path2 and path1 are internal variables to the method. They shouldn't be showing up in the documentation.